restapi/restapi.go

```diff
@@ -198,7 +198,30 @@ func getPersonsFromResults(peopleList []*gabs.Container, compareAttr string, des
 				continue
 			}
 
-			peep.Attributes[sourceKey] = person.Path(sourceKey).Data().(string)
+			val := person.Path(sourceKey).Data()
```

Comment:
I'm wondering if it would be worth refactoring this `getPersonsFromResults` function into two to avoid the deep nesting.